datafusion/sqllogictest/test_files/between.slt

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Would it be possible to move this file into this https://github.com/apache/datafusion/tree/main/datafusion/sqllogictest/test_files/expr directory? 

We can do it as a follow on PR too

Reply:
thanks. moved it.